pallets/liquidation/src/lib.rs

```diff
@@ -129,13 +129,45 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type BorrowingContract<T: Config> = StorageValue<_, EvmAddress, ValueQuery, DefaultBorrowingContract>;
 
+	#[pallet::validate_unsigned]
+	impl<T: Config> ValidateUnsigned for Pallet<T>
+	where
+		T::AccountId: AsRef<[u8; 32]> + IsType<AccountId32>,
+	{
+		type Call = Call<T>;
+
+		fn validate_unsigned(source: TransactionSource, call: &Self::Call) -> TransactionValidity {
+			match source {
+				TransactionSource::External => {
+					// receiving unsigned transaction from network - disallow
+					return InvalidTransaction::Call.into();
+				}
+				TransactionSource::Local => {}   // produced by off-chain worker
+				TransactionSource::InBlock => {} // some other node included it in a block
+			};
+
+			let valid_tx = |provide| {
+				ValidTransaction::with_tag_prefix("liquidate_unsigned_call")
+					.priority(1_000_000)
```

Comment:
why not max?

Reply:
oracle update has to fit before, this should be outside of range any user can do already